integrations/transportation/ryanair.js

```diff
@@ -107,12 +114,18 @@ async function getActivities(pastBookings, customerId, token) {
       departureAirportCode: k.flightInfo.Origin,
       destinationAirportCode: k.flightInfo.Destination,
     }));
-
+     
   return activities;
 }
 
 async function collect(state) {
-  const { token, customerId } = state;
+  const { 
+    token, 
+    customerId, 
+    username, 
+    password,
+  } = state;
+  await logIn(username, password);
```

Comment:
The best way to store the user info would be the following way.

* Store `username` and `password` in the state.
* Fetch `token` and `customerId` at the beginning of `collect()`
* `connect()` should just verify that the user was successfully logged in, and then store the `username` and `password` in the state.

This makes sure that the integration doesn't break if the `token` or `customerId` changes.

Reply:
Makes sense, thanks for the input! I updated it and moved it to its own PR